src/StarterSite.php

```diff
@@ -101,7 +101,12 @@ public function register_taxonomies() {}
 	 * Modify ACF Gutenberg Blocks Template Location
 	 */
 	public function acf_gutenberg_blocks_template_location() {
-		return array( 'views/organisms/blocks' );
+		$theme_dir = get_template_directory();
```

Comment:
Can we leave this in as a base case so that the "old way" continues to work and the "new way" is optional?
```suggestion
		$paths = array( 'views/organisms/blocks' );
		$theme_dir = get_template_directory();
```

Reply:
Good thought! I did exactly this on NSLIY after submitting this PR ;)